gromacs/__init__.py

```diff
@@ -242,6 +242,28 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+
+# Check if GMXRC was loaded and tries to load it otherwise. Runs GMXRC in a
```

Comment:
I'd like to make this a function. Then we can add some logic to only run it if the user wants it, we can test it separately, and we can wrap it in anice try/except.

Put it into `config.py` and call the function something like
```python
def set_gmxrc_environment(GMXRC="GMXRC"):

```